hw/ip/otbn/dv/otbnsim/sim/sim.py

```diff
@@ -64,6 +64,11 @@ def start(self, collect_stats: bool) -> None:
         self._next_insn = None
         self.state.start()
 
+    def initial_secure_wipe(self) -> None:
+        '''This is run at the start of a secure wipe after reset.'''
+        # OTBN will request a new URND value, so the model has to do the same.
+        self.state._urnd_client.request()
+
```

Comment:
As I said in the previous commit, I believe modelling the new status in the Python-land makes the most sense. We are using `write` method inside `ext_regs` to communicate back to the RTL-land with something like this:
```python 
self.state.ext_regs.write('STATUS', Status.InitialSecWipe, True)
```
and with the correct plumbing we wouldn't need the previous workaround about feeding the monitor.